sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -463,6 +464,11 @@ static int mc_probe(struct platform_device *pdev)
 		card->num_configs = ARRAY_SIZE(codec_conf) - 1;
 	}
 
+	snprintf(components_string, sizeof(components_string),
+		 "cfg-spk:%d",
+		 (sof_rt711_rt1308_rt715_quirk & SOF_SDW_MONO_SPK) ?
+			2 : 4);
```

Comment:
you can merge these two lines, it'll only be 76 characters, even if you keep those redundant parentheses ;-)